src/representations/representation_manager_spherical_expansion.hh

```diff
@@ -0,0 +1,704 @@
+/**
```

Comment:
Delete this file?

Reply:
oh, more merge fallout. When will it end? :worried:

Reply:
yes it should be removed